emp/emp_role_cftemplate.yaml

```diff
@@ -0,0 +1,370 @@
+              - secretsmanager:TagResource
+            Effect: Allow
+            Resource:
+              - arn:*:secretsmanager:*:*:secret:aws.cluster.x-k8s.io/*
+        Version: 2012-10-17
+    Type: AWS::IAM::ManagedPolicy
+  # PF9 Role and Policies
+  EKSRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      RoleName: !Ref RoleName
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: 'Allow'
+            Principal:
+              Service:
+                - 'ec2.amazonaws.com'
+              AWS:
+                - 'arn:aws:iam::617350239289:root'
+            Action: 'sts:AssumeRole'
+      ManagedPolicyArns:
+        - 'arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy'
+        - 'arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy'
+        - 'arn:aws:iam::aws:policy/service-role/AmazonEFSCSIDriverPolicy'
+        - !Ref AWSIAMManagedPolicyCloudProviderNodes
+        - !Ref AWSIAMManagedPolicyControllers
+  PF9EMPPolicy:
+    Properties:
+      Description: EMP PF9 Policy
+      ManagedPolicyName: !Sub "${RoleName}-emp-policy"
+      PolicyDocument:
+        Statement:
+          - Action:
+              - ec2:CreateNetworkInterfacePermission
+            Effect: Allow
+            Resource:
+              - '*'
+            Condition:
+              StringEquals:
+                ec2:ResourceTag/eks:eni:owner: eks-vpc-resource-controller
+          - Action:
+              # permissions needed for vpcpeering. This resource is created by emp-controllers, we should be able to tag
+              # it and reduce the scope of access
+              - ec2:AcceptVpcPeeringConnection
+              - ec2:CreateVpcPeeringConnection
+              - ec2:DeleteVpcPeeringConnection
+              - ec2:DescribeVpcPeeringConnections
+              # these modify a route table, of a subnet. It's not possible to constrain the access from our side
+              # this is because, we use the VPCs already created in the customer's environment by the customer.
+              # the customer is free to add restrictions according to their use case. ec2:DescribeRouteTables, is also
+              # needed by emp-server to fill in the data for emp-ui, so the restrictions here must match the one defined
+              # below.
+              - ec2:DescribeRouteTables
+              - ec2:DeleteRoute
+              - ec2:CreateRoute
+            Resource:
+              - '*'
+            Effect: Allow
+          - Action:
+              # pricing data is mostly public, it's needed for cost calculations
+              - pricing:GetProducts
+            Resource:
+              - '*'
+            Effect: Allow
+          - Action:
+              - iam:GetRole
+              - iam:GetRolePolicy
+              - iam:ListAttachedRolePolicies
+              - iam:ListRolePolicies
+              - iam:GetPolicy
+              - iam:GetPolicyVersion
+              - iam:SimulateCustomPolicy
```

Comment:
why is this needed?

Reply:
for `iam:SimulateCustomPolicy` when it's reworked